gms/impl/src/main/java/com/linkedin/metadata/resources/entity/AspectResource.java

```diff
@@ -40,7 +38,7 @@
       @QueryParam("aspect") @Optional @Nullable String aspectName,
       @QueryParam("version") @Optional @Nullable Long version
       ) throws URISyntaxException {
-    _logger.info("GET ASPECT urn: {} aspect: {} version: {}", urnStr, aspectName, version);
+    log.info("GET ASPECT urn: {} aspect: {} version: {}", urnStr, aspectName, version);
```

Comment:
Should we change this to debug? These logs have been very helpful for debugging tho. A bit torn.

Reply:
I know... I think in light of having the new LoggingFilter we should be able to debuggify these. Let's do this in a followup since this isn't critical